### ISSUE TITLE
Error unknown room status PERM_FAILURE after migrating to go bridge

### ISSUE THREAD

**Issue by JuliusFreudenberger** [mautrix/signal#460]
I waited with the migration to the go bridge up until now. I have a dockerized setup and did a pull to the latest docker image at dock.mau.dev/mautrix/signal (Now I'm running the image with digest sha256:d877923e2bb791632cb94fb5b1e590e8574f8dc031c38dd73b1b50d3538ded21). I did not change anything else in the setup.

As to be expected, I can not write and receive messages to Signal Bridge rooms. However, the Signal Bridge Bot does not respond either. Below is an excerpt of the logs from the bridge when sending a message (I redacted some values but can provide these if needed). I think the last line is quite interesting, but I couldn't find any information about this in past issues or PRs. I did not change the registration or keys during the migration.
I also created a new management room which sadly did not help.

```
2024-02-21T23:42:49Z DBG Starting handling of transaction content={"pdu":0,"unstable_edu":1,"unstable_to_device":0} transaction_id=225300
2024-02-21T23:42:49Z DBG Finished dispatching events from transaction transaction_id=225300
2024-02-21T23:42:49Z DBG Starting handling of transaction content={"pdu":1,"unstable_edu":0,"unstable_to_device":0} transaction_id=225301
2024-02-21T23:42:49Z DBG Finished dispatching events from transaction transaction_id=225301
2024-02-21T23:42:49Z DBG Decrypting received event event_id=<redacted> session_id=<redacted>
2024-02-21T23:42:49Z DBG Sent message checkpoint message_checkpoint={"event_id":"<redacted>","event_type":"m.room.encrypted","reported_by":"BRIDGE","retry_num":0,"room_id":"<redacted>:jfreudenberger.de","status":"SUCCESS","step":"BRIDGE","timestamp":1708558969399}
2024-02-21T23:42:49Z DBG Ratchet safety data changed (ratchet state didn't change) action="decrypt megolm event" event_id=<redacted> lost_indices=[] max_messages=100 missed_indices=[0] new_ratchet_index=0 next_new_index=2 prev_ratchet_index=0 sender=@<sendername>:jfreudenberger.de sender_key=<redacted>
2024-02-21T23:42:49Z DBG Event decrypted successfully action="decrypt megolm event" decrypted_event_type="m.room.message (message)" event_id=<redacted> message_index=1 sender=@<sendername>:jfreudenberger.de sender_key=<redacted> session_id=<redacted>
2024-02-21T23:42:49Z DBG Sent message checkpoint message_checkpoint={"event_id":"<redacted>","event_type":"m.room.message","message_type":"m.text","reported_by":"BRIDGE","retry_num":0,"room_id":"<redacted>:jfreudenberger.de","status":"SUCCESS","step":"DECRYPTED","timestamp":1708558969408}
2024-02-21T23:42:49Z DBG Sent message checkpoint message_checkpoint={"event_id":"<redacted>","event_type":"m.room.message","info":"unknown room","message_type":"m.text","reported_by":"BRIDGE","retry_num":0,"room_id":"<redacted>:jfreudenberger.de","status":"PERM_FAILURE","step":"REMOTE","timestamp":1708558969409}
```
I can provide further logs or more detail about the setup if needed.

**Comment by richard-kramer** [comment]
I've got the same problem. Bridging signal messages does not work and the bridgetbot doesn't respond either.

**Comment by JuliusFreudenberger** [comment]
I played around a bit more and could get it working again by prefixing the commands with `!signal`. (Yay!)

Before upgrading, this however was my management room. The bridge seems to have forgotten that this was the management room. (It also says so in the top of the help message: "This is not your management room: prefixing commands with !signal is required.") Also the other rooms I created are not recognized as such. Maybe this information is not migrated correctly?
